src/signals/incident/services/custom-validators/custom-validators.ts

```diff
@@ -30,6 +29,21 @@ export const falsyOrNumber = (control: Control) => {
   }
 }
 
+export const inPast = (control: Control) => {
+  const newDate = new Date()
+
+  if (
+    !control ||
+    !control.value ||
+    parseInt(control.value) <= newDate.getTime()
```

Comment:
Since control is mandatory and with the above type change you know what the type is it will be sufficient to just check `parseInt(control.value) <= newDate.getTime()`